views/login.pug

```diff
@@ -0,0 +1,22 @@
+extends layout 
+block content 
+    h1 Hey there guy. Login
+    form(action='http://127.0.0.1:3000/login' method='POST')
```

Comment:
Use a relative path here: `/login`, so that your form action is not hardcoded to localhost when you get ready to deploy